docs/internals/optimizer.rst

```diff
@@ -1117,9 +1168,9 @@ and replace all references to ``f`` by ``f2``.
 The inliner should be run afterwards to make sure that all references to ``f2`` are replaced by
 ``f``.
```

Comment:
I assumed this means `ExpressionInliner`. Is that correct?

Reply:
No, that's just coincidence here - if you have a function with three return arguments and only one of them is unused, so two remain, only the full inliner can inline this.

Reply:
Updated to `FullInliner`.